src/chapters/sql-part2/creating-databases.rst

```diff
@@ -0,0 +1,115 @@
+``pen_drawer`` will require foreign key columns. In the query tab of MySQL
+Workbench, ``CREATE`` the tables as described below.
+
+``writing_supply``
+^^^^^^^^^^^^^^^^^^^
+
+3. Each record in this table will have three fields:
+
+   a. A primary key, ``supply_id``.
+   #. A ``utensil_type``, which will be an ``ENUM`` ("Pencil" or "Pen").
+   #. An integer number of drawers, ``num_drawers``.
+
+``pencil_drawer``
+^^^^^^^^^^^^^^^^^
+
+4. Each record in this table will have five fields:
+
+   a. A primary key, ``drawer_id``.
+   b. The ``pencil_type``, which will be an ``ENUM`` ("Wood" or "Mechanical").
+   c. An integer number of pencils, ``quantity``.
+   d. A boolean describing whether or not it is time to ``refill`` the drawer.
+   e. A foreign key (``supply_id``) that connects ``pencil_drawer`` with
+      ``writing_supply``.
+
+``pen_drawer``
+^^^^^^^^^^^^^^
+
+5. Each record in this table will have five fields:
+
+   a. A primary key, ``drawer_id``.
+   b. The ``color`` of the pens, which will be an ``ENUM`` ("Black", "Blue",
+      "Red", "Green", "Purple").
+   c. An integer number of pens, ``quantity``.
+   d. A boolean describing whether or not it is time to ``refill`` the drawer.
+   e. A foreign key (``supply_id``) that connects ``pen_drawer`` with
+      ``writing_supply``.
+
+Import Data
+-----------
+
+.. admonition:: Note
+
+   Whenever you import existing data into your empty tables, it is helpful to
+   use the same column names as the external source.
+
+#. Follow this link to the
+   `practice data <https://gist.github.com/jimflores5/795a7faae8d35d36033872ae30b88328>`__.
+#. As you did in the first :ref:`SQL studio <movie-sqls>`, download the zip
+   file. Double-click on it to extract three ``.csv`` files.
+#. In MySQL Workbench, right-click on the ``writing_supply`` table. Select
+   *Table Data Import Wizard*.
+
+   .. figure:: ./figures/SQLWorkbenchImport.png
+      :alt: Table Data Import Wizard menu option.
+
+#. In the next panel, click the *Browse* button. Find and select the
+   ``writing_supply.csv`` file. Click *Next*.
+
+   .. figure:: ./figures/importCsvFile.png
+      :alt: Select file path to csv file.
+
+#. In the following panel, use the default option
+   *Use existing table --> storage.writing_supply*, and click *Next*.
+#. Once again, accept the default options for importing data into your table
+   columns. Click *Next*.
+
+   .. figure:: ./figures/importColumns.png
+      :alt: Select columns to import.
+
+#. Clicking *Next* again will import the data. Clicking *Finish* on the final
+   panel returns you to the editor.
+#. In the query tab, run ``SELECT * FROM writing_supply`` to confirm that 6
+   entries now exist in the table.
+#. Repeat the steps 3 - 8 for the ``pencil_drawer`` and ``pen_drawer`` tables.
+
+Confirm that the ``pencil_drawer`` and ``pen_drawer`` tables hold 6 and 15
+entries, respectively. You are now ready to practice more advanced SQL queries.
+
```

Comment:
Sometimes the CSV import doesn't work properly. Perhaps add a warning with some steps students can take if this process goes wrong?

Reply:
I added a troubleshooting section.  Let me know what you think.